src/Lean/Elab/Command.lean

```diff
@@ -503,6 +504,45 @@ def expandDeclId (declId : Syntax) (modifiers : Modifiers) : CommandElabM Expand
 
 end Elab.Command
 
+open Elab Command MonadRecDepth
+
+/--
+Lift an action in `CommandElabM` into `CoreM`, updating the traces and the environment.
+This does not preserve things like `open` and `namespace` declarations.
+-/
+def liftCommandElabM (k : CommandElabM α) : CoreM α := do
```

Comment:
The parameter name `k` suggests a continuation, but this is not that. What about calling it `commands` instead?